### PR TITLE
Updated the Free-Hosting Section Position

### PR DESCRIPTION
**Pull request by GundaSudarrshan**
I have updated the Free-Hosting Section to its Proper Category in the Contents section which is F-Category according to alphabet order

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -500,6 +500,19 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :green_book: [Write your first Flutter app, Part 2](https://codelabs.developers.google.com/codelabs/first-flutter-app-pt2/#0)
 - :green_book: [Write a Flutter desktop application](https://codelabs.developers.google.com/codelabs/flutter-github-graphql-client/#0)
 
+---
+**[⬆ Back to Index](#content)**
```
Comment (shahednasser) [inline README.md:504]:
```suggestion
---

**[⬆ Back to Index](#content)**
```

File: README.md [hunk 2]
```diff
@@ -574,19 +587,7 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
...
-
-## Free Hosting
-
-- :wrench: [Netlify](https://www.netlify.com/)
-- :wrench: [Heroku](https://www.heroku.com/)
-- :wrench: [Vercel](https://vercel.com/)
-- :wrench: [Github Pages](http://pages.github.com)
-- :wrench: [Gitlab Pages](http://docs.gitlab.com/ee/user/project/pages)
-- :wrench: [Deta](https://www.deta.sh/)
-- :wrench: [InfinityFree](https://infinityfree.net/)
```
Comment (shahednasser) [inline README.md:590]: Remove this line

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (shahednasser) [reply README.md:599]: There are many empty lines after the `---` now that the section is moved. There should only be one empty line after it

Comment (GundaSudarrshan) [reply README.md]: I have resolved it please check and update

**Pull request merged**